kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMConfigurationServiceImpl.java

```diff
@@ -295,6 +313,77 @@ private void decryptAndConvertPasswordProperties(Map<String, Object> modifiedPro
         }
     }
 
+    private void decryptAndConvertCertificatesProperties(Map<String, Object> modifiedProps) throws KuraException {
```

Comment:
Remove the declaration of thrown exception 'org.eclipse.kura.KuraException', as it cannot be thrown from method's body.

```suggestion
    private void decryptAndConvertCertificatesProperties(Map<String, Object> modifiedProps){
```